pkg/web/router.go

```diff
@@ -0,0 +1,41 @@
+package web
```

Comment:
I think it would be best to give the router its own package. So it could be called like:

```go
r := router.New()
```

It could be added under **pkg/web/router**.